experimental/PropertyDDS/packages/property-changeset/src/changeset_operations/indexedCollection.ts

```diff
@@ -509,11 +509,9 @@ export namespace ChangeSetIndexedCollectionFunctions {
 
 				// Store the ChangeSet
 				if (in_changePrefix === "other") {
-					if (!Array.isArray(in_collection)) {
-						changesByKeys[key].change = in_collection[key];
-					} else {
-						changesByKeys[key].change = key;
-					}
+					changesByKeys[key].change = !Array.isArray(in_collection)
+						? in_collection[key]
+						: key;
```

Comment:
I was going to skip this because I know it was a mechanical replacement, but given @Josmithr 's [comment](https://github.com/microsoft/FluidFramework/pull/17380/files#r1330813547), should we also flip this one to avoid the negation?